### DISCUSSION TITLE
Quick Start & Debian 10

### DISCUSSION THREAD

**Discussion by nabberuk** [swizzin/swizzin#571] [Q&A + Help] [answered]
I've just done a clean install of Debian 10 (netboot) with system utilities and the ssh server. 

Running either of the quick start commands results in the following error (after entering a user/pass).

In this example i run, bash <(curl -sL git.io/swizzin) && . ~/.bashrc. I presume the file exists error is due to this being the second times i've ran the command.

Updating system and grabbing core dependencies.
Performing a system upgrade
Installing dependencies
Cloning swizzin repo to localhost
ln: failed to create symbolic link '/usr/local/bin/swizzin/scripts': File exists
Creating new user username ...
/dev/fd/63: line 190: useradd: command not found

I've double checked and adduser is installed

**Comment by nabberuk** [comment, marked as answer]
ah, sorted it by following:

https://devconnected.com/how-to-add-and-delete-users-on-debian-10-buster/#:~:text=the%20commands%20above.-,adduser%20%3A%20command%20not%20found%20on%20Debian,sbin%E2%80%9D%20folder%20of%20your%20system.&text=To%20solve%20this%20issue%2C%20you,sbin%E2%80%9D%20to%20your%20%24PATH.&text=Source%20your%20bashrc%20file%20and%20try%20to%20run%20the%20adduser%20command%20again.

**Reply by flying-sausages** [reply]
Can you let us know which commands specifically helped you out?

**Reply by nabberuk** [reply]
Of course, i added the following to ~/.bashrc 

So i did:
`vi ~/.bashrc`

Added:
`export PATH="$PATH:/usr/sbin/"`

Saved the file and then ran:
`source ~/.bashrc`

You'll know if its now going to work as you should be able to run adduser without error
